docs/the-build-pipeline.md

```diff
@@ -1,11 +1,13 @@
-# The Build Pipeline
+# Build Pipeline and Web Deployment
 
 ownCloud uses <a href="https://drone.io/">Drone</a> for building and deploying the documentation.
 Drone is a [Continuous Delivery](https://www.continuousdelivery.com/) platform that helps optimize and automate software delivery.
 
 The build pipeline is configured in [.drone.yml](https://github.com/owncloud/docs/blob/master/.drone.yml), located in the root directory of the docs repository.
 You can view the current build status of the docs at https://drone.owncloud.com/owncloud/docs.
 
+Every push to the docs master branch triggers a drone build. At the end of the build process, the docs are getting deployed on the web.
```

Comment:
s/the docs are getting deployed on the web/the docs are deployed to production/g